website/common/script/ops/buy/buySpecialSpell.js

```diff
@@ -25,6 +25,16 @@ module.exports = function buySpecialSpell (user, req = {}, analytics) {
 
   user.items.special[key] += quantity;
 
+  if (key === 'salt') {
```

Comment:
the debuffs needs to be executed by casting a spell instead of buying it (yes in the background it still is bought, but a different api call)

you'd need to change `customPurchase` inorder to call `this.castStart(..)` in `app.vue`